aws/dynamodb.go

```diff
@@ -0,0 +1,71 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/gruntwork-io/cloud-nuke/logging"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+)
+
+func getAllDynamoTables(session *session.Session) ([]*string, error) {
+	var TableNames []*string
+	svc := dynamodb.New(session)
+
+	for {
+		result, err := svc.ListTables(&dynamodb.ListTablesInput{})
```

Comment:
Does this API call need pagination?

Reply:
Bump

Reply:
Bumping this one too. It would be best to have pagination in place, since it's already supported by default in the ListTables API method too. The maximum number of DynamoDB table names returned seems to be 100 per query. https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_ListTables.html

Could we add that?